lib/fencing/st_client.c

```diff
@@ -1757,17 +1762,21 @@ stonith_api_validate(stonith_t *st, int call_options, const char *rsc_id,
         *error_output = NULL;
     }
 
+    if (timeout_sec <= 0) {
+        timeout_sec = CRMD_METADATA_CALL_TIMEOUT; // Questionable
```

Comment:
Maybe `CRMD_METADATA_CALL_TIMEOUT + crm_get_msec(CRM_DEFAULT_OP_TIMEOUT_S)`? `stonith__rhcs_validate()` contains a metadata action and a validate-all action.